### DISCUSSION TITLE
database update message

### DISCUSSION THREAD

**Discussion by jimsaville** [burnsoftnet/burnsoftnet.github.io#4] [General]
Every time the software is used since updating to the latest version, the database update message appears.  As soon as one clicks on anything which accesses the database the sequence appears:  Database Updates Required, Updates need to be applied, Apply Now?  Click on the "Yes" button, software indicates Applied Hotfix: 10.  Click on "OK" button and you can continue to use software.

**Comment by burnsoftnet** [comment]
Which Version did you have before?
When you click on Yes and it is all done, Is there a Red Icon at the bottom left hand corner of the Main Window?
If there is a Red Icon, can you double click on it and send in the results of that file?  Or at least the lines with the current date and time.

**Comment by jimsaville** [comment]
Previous version was v6.5.0.1.  I believe it was the original open source release.

There is no red icon in any window.

**Reply by burnsoftnet** [reply]
Thanks for the information.  I'll try to replicate the issue and hopefully have a fix for this and the password issue this week.  New Process of upgrading the database from the previous versions was bound to have a glitch somewhere.

**Comment by jimsaville** [comment]
Agreed.  Issues are inevitable and are why we test.  Thanks for your
assistance.

On Tue, Sep 6, 2022 at 1:53 PM BurnSoft ***@***.***> wrote:

> Thanks for the information. I'll try to replicate the issue and hopefully
> have a fix for this and the password issue this week. New Process of
> upgrading the database from the previous versions was bound to have a
> glitch somewhere.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/burnsoftnet/burnsoftnet.github.io/discussions/4#discussioncomment-3578007>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A24CY7UU5U3FDVO4XCSPKLTV46ALNANCNFSM6AAAAAAQFCBSTM>
> .
> You are receiving this because you authored the thread.Message ID:
> <burnsoftnet/burnsoftnet.github.io/repo-discussions/4/comments/3578007@
> github.com>
>


-- 
Jim Saville
Kaleidoscope Systems
301-346-5406
***@***.***

"The game of life is the game of boomerangs. Our thoughts, deeds and words
return to us sooner or later, with astounding accuracy."
- Florence Shinn -

**Reply by burnsoftnet** [reply]
New Release v6.9.15 Should resolve this issue.  https://github.com/burnsoftnet/MyGunCollection/releases/tag/v6.9.15.2
